### ISSUE TITLE
Compatibility with sonarqube community 6.7

### ISSUE THREAD

**Issue by bjoernhaeuser** [InfoSec812/sonar-auth-google#8]
Hi there,

I just tried this plugin with sonarqube 6.7 community edition. 

Unfortunately the "Security" -> "Google" does not show up. Even if I set the property values in sonar.properties file, the login button does not show up.

Is there any log file I can look into?

Thanks!
Björn

**Comment by InfoSec812** [comment]
I'll take a look at it as soon as I can

**Comment by bjoernhaeuser** [comment]
Happy to help with everything I can do. Though I have no idea how plugins in sonarqube work :(

**Comment by InfoSec812** [comment]
Confirmed. In 6.5 they deprecated the `Settings` API and replaced it with the `Configuration` API... It will require a little work to grok the new API and implement.

**Comment by InfoSec812** [comment]
It appears that it should still work OK until version 6.6, but 6.7 has breaking changes... For now, I would recommend downgrading to 6.6. I will try to figure out the new API sometime this week while there is a lull at work.

**Comment by giannello** [comment]
The configuration is actually available under "Configuration -> General settings" after selecting "Security" in the left sidebar

![image](https://user-images.githubusercontent.com/551789/33325997-0752b62e-d454-11e7-996a-e17ef0824732.png)

**Comment by InfoSec812** [comment]
@giannello Does this mean that you have tried it and it is working in 6.7? It looks like there are breaking changes in the configuration API in 6.7, so I was pretty certain it wouldn't work.

**Comment by giannello** [comment]
@InfoSec812 Yes, I've just deployed a fresh install of 6.7, and - after reading this ticket and accidentally finding the configuration in the right place - it works just fine. I have my google users created in the database, I can login/logout and do the usual stuff without apparent problems. Not sure tho if I'm missing something, as this is my first SQ deployment with your plugin.

**Comment by InfoSec812** [comment]
@giannello Awesome! Thanks for the feedback... If you can log on with your GApps account, that's all I am concerned with. I will still be taking a look at the changes WRT 6.7 and configuration, but now I feel like it's not quite as high of a priority... Thanks!!!
